service/handlers.go

```diff
@@ -604,14 +604,25 @@ func (h handler) getProject(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	level.Debug(l).Log("message", "getting project")
+	level.Debug(l).Log("message", "getting project from credentials provider")
 	resp, err := cp.GetProject(projectName)
```

Comment:
One thing worth considering here is that we do this call essentially just to check the project exists. We may want to refactor/replace this GetProject call to not return the Response object

Reply:
I think for the future we may want this to continue returning the project data as you'd expect, even if we aren't using the data now.